core/src/main/scala/io/chrisdavenport/npmpackage/PackageFile.scala

```diff
@@ -24,6 +24,7 @@ object PackageFile {
     additionalNpmConfig: Map[String, Json],
     enableBinary: Boolean,
     binaryArtifacts: Seq[(String, String)],
+    `type`: String,
```

Comment:
Indeed, this is theoretically a breaking change ... but if you just add a deprecated version of this method without this parameter that passes the old CommonJS default I'd say we are golden.

Reply:
Done. I've put the `since` to `0.1.1`, but I don't know what you'd want to release this under